README.md

```diff
@@ -1 +1,29 @@
 # StardewValleyPocketTracker
+## Purpose
```

Comment:
A horizontal rule between the top level heading and the Purpose heading would be nice